src/core/index.js

```diff
@@ -228,14 +102,14 @@ export default function SwaggerUI(opts) {
     return system
   }
 
-  const configUrl = queryConfig.config || constructorConfig.configUrl
+  const configUrl = combinedConfig.configUrl
```

Comment:
The name for this option is `configUrl` and not like we were trying to get `queryConfig.config`. We also shouldn't need to do that, as the option from `queryConfig` should take precedence after being merged into `combinedConfig`, so it looks to me like this should be enough here.

Reply:
We'll create a separate issue for this one